components/FoodGroups/foodgroups.module.css

```diff
@@ -0,0 +1,20 @@
+/**********/
+/* Five food groups main page */
+/**********/
+
+:root .dairy,
+.meat,
+.fruit,
+.vegetables,
+.grains {
+  grid-row-start: 1;
+  grid-column-start: 1;
+}
+
+.wrapper-fix > span {
+  position: relative !important;
+}
+
+.wiggle-animate {
+  animation-delay: -250ms;
+}
```

Comment:
consider converting this to tailwind

Reply:
`.wrapper-fix > span {
  position: relative !important;
}`

is an override to the Image Component wrapper span, since there is no way to use child styling in tailwind to my knowledge.

The other stylings are for animations

Reply:
I have added a tailwind class in `main.css` which is 
```css
@layer utilities {
    .span-child-relative > span {
      position: relative;
    }
}
```
so you can replace `wrapper-fix` with `!span-child-relative`. The `!` adds the `!important` tag to the css property.